spec/system/polls/polls_spec.rb

```diff
@@ -401,6 +414,19 @@
         expect(page).to have_link("Yes")
       end
     end
+
+    scenario "Shows SDG tags when feature is enabled" do
+      Setting["feature.sdg"] = true
+      Setting["sdg.process.polls"] = true
+
+      poll = create(:poll, sdg_goals: [SDG::Goal[1]],
+                           sdg_targets: [SDG::Target["1.1"]])
+
+      visit poll_path(poll)
+
+      expect(page).to have_selector(".sdg-goal-tag-list")
+      expect(page).to have_selector(".sdg-target-tag-list")
```

Comment:
How about testing it from the user's point of view?

```suggestion
      expect(page).to have_selector "img[alt='1. No Poverty']"
      expect(page).to have_content "target 1.1"
```